src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -244,18 +251,19 @@ private static String firstImpExtLanguage(Collection<ExtImpRubicon> rubiconImpEx
                 .orElse(null);
     }
 
-    private BidRequest createSingleRequest(Imp imp, ExtImpRubicon rubiconImpExt, BidRequest bidRequest,
-                                           String impLanguage, boolean useFirstPartyData) {
+    private BidRequest createSingleRequest(Imp imp, ExtPrebid<ExtImpPrebid, ExtImpRubicon> ext,
```

Comment:
Minor. For better readability maybe it make sense to split `ExtPrebid ext` into 2 separate arguments: `ExtImpPrebid extImpPrebid` and `ExtImpRubicon extImpRubicon`.
What do you think?